test/clj/rems/test/api/forms.clj

```diff
@@ -86,6 +86,53 @@
                         :items
                         (map #(select-keys % [:optional :type :localizations])))))))))))
 
+(deftest option-form-item-test
+  (let [api-key "42"
+        user-id "owner"]
+    (testing "create"
+      (let [command {:organization "abc"
+                     :title (str "form title " (UUID/randomUUID))
+                     :items [{:title {:en "en title"
+                                      :fi "fi title"}
+                              :optional true
+                              :type "option"
+                              :options [{:key "yes"
+                                         :label {:en "Yes"
+                                                 :fi "Kyllä"}}
+                                        {:key "no"
+                                         :label {:en "No"
+                                                 :fi "Ei"}}]}]}
+            response (-> (request :post "/api/forms/create")
+                         (authenticate api-key user-id)
+                         (json-body command)
+                         app)]
+        (assert-response-is-ok response)
```

Comment:
You can now move this row to the `(-> ... assert-response-is-ok)` since master contains the chainable version of the assert.